README.md

```diff
@@ -93,25 +93,25 @@ Note: only supports Linux; no Windows or MacOS.
     ```
     Build your docker container using: `docker build . --file Objectnav.Dockerfile  -t objectnav_submission`. (Note: you may need `sudo` priviliges to run this command.)
 
-1. b) ObjectNav: Download Habitat-Matterport3D Dataset scenes used for Habitat Challenge [following instructions here](https://github.com/facebookresearch/habitat-sim/blob/main/DATASETS.md#habitat-matterport-3d-research-dataset-hm3d). Place this data in: `habitat-challenge/habitat-challenge-data/data/scene_datasets/hm3d`
+1. Dataset: First, get access to the Habitat-Matterport3D Dataset scenes by visiting [this link](https://matterport.com/habitat-matterport-3d-research-dataset) and following the given instructions. After getting access to the dataset, carry out the following steps to download the dataset:
+    
+    a) First, you will need to generate a matterport API Token:
+
+    1. Navigate to https://my.matterport.com/settings/account/devtools
+        
+    1. Generate an API token
+        
+    1. Your API token ID then functions as your username, passed to the download script with --username, and your API token secret functions as your password, passed to the download script with --password. Note: Make sure to write your API token secret down, you can't reveal it again!
+
+    b) Now, you are ready to download. Start by downloading the val split, which we will use in the following steps:
 
-    **Using Symlinks:**  If you used symlinks (i.e. `ln -s`) to link to an existing download of HM3D, there is an additional step. First, make sure there is only one level of symlink (instead of a symlink to a symlink link to a .... symlink) with
       ```bash
-      ln -f -s $(realpath habitat-challenge-data/data/scene_datasets/hm3d) \
-          habitat-challenge-data/data/scene_datasets/hm3d
+      python -m habitat_sim.utils.datasets_download --username <api-token-id> --password <api-token-secret> --uids hm3d_val --data-path <path to download folder>
       ```
-
-     Then modify the docker command `test_locally_objectnav_rgbd` to mount the linked to location by adding
-     `-v $(realpath habitat-challenge-data/data/scene_datasets/hm3d):/habitat-challenge-data/data/scene_datasets/hm3d`.  The modified docker command
-     would be
-     ```bash
-    docker run \
-          -v $(pwd)/habitat-challenge-data:/habitat-challenge-data \
-          -v $(realpath habitat-challenge-data/data/scene_datasets/hm3d):/habitat-challenge-data/data/scene_datasets/hm3d \
-          --runtime=nvidia \
-          -e "AGENT_EVALUATION_TYPE=local" \
-          -e "TRACK_CONFIG_FILE=/challenge_objectnav2022.local.rgbd.yaml" \
-          ${DOCKER_NAME}
```

Comment:
Are you sure ^these lines can be removed? Our HM3D download contains a link. 

When I evaluate docker locally, I see an error: 

```
Traceback (most recent call last):
  File "agent.py", line 41, in <module>
    main()
  File "agent.py", line 33, in main
    challenge = habitat.Challenge(eval_remote=False)
  File "/habitat-lab/habitat/core/challenge.py", line 16, in __init__
    super().__init__(config_paths, eval_remote=eval_remote)
  File "/habitat-lab/habitat/core/benchmark.py", line 38, in __init__
    self._env = Env(config=config_env)
  File "/habitat-lab/habitat/core/env.py", line 108, in __init__
    id_sim=self._config.SIMULATOR.TYPE, config=self._config.SIMULATOR
  File "/habitat-lab/habitat/sims/registration.py", line 19, in make_sim
    return _sim(**kwargs)
  File "/habitat-lab/habitat/sims/habitat_simulator/habitat_simulator.py", line 271, in __init__
    super().__init__(self.sim_config)
  File "<attrs generated init habitat_sim.simulator.Simulator>", line 11, in __init__
  File "/opt/conda/envs/habitat/lib/python3.7/site-packages/habitat_sim/simulator.py", line 122, in __attrs_post_init__
    self.__set_from_config(self.config)
  File "/opt/conda/envs/habitat/lib/python3.7/site-packages/habitat_sim/simulator.py", line 260, in __set_from_config
    self._config_backend(config)
  File "/opt/conda/envs/habitat/lib/python3.7/site-packages/habitat_sim/simulator.py", line 204, in _config_backend
    super().__init__(config.sim_cfg, config.metadata_mediator)
AssertionError: ESP_CHECK failed: Missing (at least) one of scene dataset attributes, stage attributes, or dataset scene attributes for scene 'habitat-challenge-data/data/scene_datasets/hm3d/val/00802-wcojb4TFT35/wcojb4TFT35.basis.glb'.  Likely an invalid scene name.
(habitat-conda) dbatra@ubuntu-bionic-3:~/habitat-challenge$ ls habitat-challenge-data/data/scene_datasets/hm3d/val/00802-wcojb4TFT35/wcojb4TFT35.basis.glb
habitat-challenge-data/data/scene_datasets/hm3d/val/00802-wcojb4TFT35/wcojb4TFT35.basis.glb
```

Reply:
Confirming that they can be deleted.